package-res/resources/web/pentaho/data/filter/AbstractPropertyFilter.js

```diff
@@ -63,6 +64,9 @@ define([
    *   var filteredData = filter.apply(data); //filteredData.getValue(0, 0) === "A"
    * });
    *
+   * @description The `AbstractPropertyFilter` extends the `IAbstractFilter` interface
+   * to allow creation of types of filters for a data table.
+   *
```

Comment:
If I lookup  IAbstractFilter, I get: 

> The `IAbstractFilter` interface allows creation of other types of filters for a data table.

So an `AbstractPropertyFilter` extends the  capability of creating other types of filters for a data table with the capability of creating types of filters for a data table.
:-1: